src/pages/components/button/usage.mdx

```diff
@@ -102,186 +132,419 @@ import { Add16 } from "@carbon/icons-react";
+| _Full bleed_ | 64/4            | Use when buttons bleed to the edge of a larger component, like side panels or modals. _Note: This variant is not offered in the kit because it does not have a coded counterpart. Carbon only offers this button in the context of the [modal component](components/modal/usage)._ |
+| _Default_    | 48/3            | Use as primary page actions and other standalone actions.                                                                                                                                                                                                                          |
+| _Field_      | 40/2.5          | Use when buttons are paired with input fields.                                                                                                                                                                                                                                     |
+| _Small_      | 32/2            | Use when there is not enough vertical space for the default or field-sized button.                                                                                                                                                                                                 |
+
+### Emphasis
+
+You don’t necessarily need to use the buttons in the order that their labels imply. For example, you don’t always need to use the secondary button as the second button in your layout. The most important thing is to establish a visual hierarchy between the buttons in your UI. Keep these best practices in mind.
+
+#### A single, high-emphasis button
+
+As a general rule, a layout should contain a single high-emphasis button that makes it clear that other buttons have less importance in the hierarchy. This high-emphasis button commands the most attention.
+
+#### Multiple button emphasis
+
+A high-emphasis button can be accompanied by medium- and low-emphasis buttons that perform less important actions. Keep in mind that you should only group together calls to action that have a relationship to one another.
+
+Although secondary buttons have less visual prominence because they are less saturated than their primary counterparts, they are still tonally heavy. If your layout requires multiple actions—as is the case with some toolbars, data lists and dashboards—low emphasis buttons (tertiary or ghost) may be a better choice.
+
 <Row>
 <Column colLg={8}>
 
-![Secondary / Primary button order in a modal](images/button-usage-4.png)
+![Button emphasis](images/button_usage_7.png)
+
+</Column>
+</Row>
+
+<DoDontRow>
+  <DoDont caption="Do use high-emphasis and medium-emphasis buttons in a button group.">
+
+![Do use high-emphasis and medium-emphasis buttons in a button group.](images/button_usage_8_test.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not center text in a button, even if the button is wide.">
```

Comment:
I think the "Don't" caption here refers to something else; seems like it should be "don't use more than one Primary button".
![image](https://user-images.githubusercontent.com/35537391/85622530-f6990700-b634-11ea-88d1-7fc1721d1f83.png)

Reply:
Thanks Con Bon!!! Good catch!